src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private Color color;
```

Comment:
You already have `Color color` field in `Figure` class.

Reply:
It has no sence, cuz it has to be inharited from Figure. Changed that  everywhere